### PR TITLE
String to list based converters

### PR DESCRIPTION
**Pull request by bijington**
Documentation added for .NET MAUI Community Toolkit proposals:

- [Proposal] `ListToStringConverter` <https://github.com/CommunityToolkit/Maui/issues/70> includes:

- [Proposal] `StringToListConverter ` <https://github.com/CommunityToolkit/Maui/issues/75> includes:

### CODE REVIEW

**Comment by PRMerger15** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by bijington** [comment]
@brminnick I would appreciate a code review of this file if you wouldn't mind:

docs/maui/converters/string-to-list-converter.md

File: docs/maui/converters/string-to-list-converter.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
+---
```
Comment (bijington) [inline docs/maui/converters/string-to-list-converter.md:1]: @brminnick lets try that again, this is the file I would like you to take a look over given the more complex markup scenario.

**Comment by PRMerger6** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger6** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger8** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger18** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger18** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Review by brminnick — APPROVED** [review]

**Pull request merged**